src/kgcl_schema/grammar/parser.py

```diff
@@ -54,6 +58,15 @@ def parse_statement(input: str) -> Change:
 
     Return an instantiated dataclass object from model.kgcl_schema.
     """
+    regex = r'<http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+'
```

Comment:
This does not seem to end with `>`, but see comments below.

Reply:
Resolved.